### PR TITLE
Adding a devices tab for references

### PR DESCRIPTION
**Pull request by amma35**
Adding a devices tab for references see #37

### CODE REVIEW

**Comment by amma35** [comment]
On the question raised in the old PR: 
Regarding the import device, you shoulds only be Proposed That the devices can be connected to an item (for example you can not add a peripheral was soundcard). THUS, once the device is created, infocom are not filled (as generated for the item)

I fixed :  propose devices that can be connected to an item

File: inc/item_devices.class.php [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                  action='".$CFG_GLPI['root_doc']."/front/item_devices.form.php' method='post'>\n";
+         echo "\t<input type='hidden' name='items_id' value='$ID'>\n";
+         echo "\t<input type='hidden' name='itemtype' value='".$item->getType()."'>\n";
+      }
+
+      $table->display(array('display_super_for_each_group' => false,
+                            'display_title_for_each_group' => false));
+
+      if ($canedit) {
+//          echo "<input type='submit' class='submit' name='updateall' value='" .
```
Comment (trasher) [inline inc/item_devices.class.php]: Please remove comments

Comment (amma35) [inline inc/item_devices.class.php]: OK

**Comment by trasher** [comment]
Please rebase to fix conflitcs.

Also bump minimal GLPI required version.

**Pull request closed without merging**